docs/helm/eks-deployment.md

```diff
@@ -128,7 +160,9 @@ Now we have an EKS cluster up and running there are a few one time steps we need
 
     ![Attach Policy](./diagrams/eks-attach-policy.png)
 
-### File System
+### Storage
+
```

Comment:
Again for storage I think we need to make it clear this is just one example and one can also use S3 with the connector (linking to tthe aws-services doc) or even a local block device in a non-cluster env.